### PR TITLE
Multiple FE WEB UI login issues

### PR DESCRIPTION
**Pull request by hf200012**
Multiple FE WEB UI login issues

## Proposed changes

Close related #issue (replace it with issue number if it exists).

Describe the overview of changes, and introduce why we need it.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/faq/faq.md [hunk 1]
```diff
@@ -318,6 +318,79 @@ failed to initialize storage reader. tablet=63416.1050661139.aa4d304e7a7aff9c-f0
...
 
+### Q22. 多个FE，在使用Nginx实现web UI负载均衡的时候，登录不进去的情况
+
+Doris 可以部署多个FE，在访问Web UI的时候，如果使用Nginx进行负载均衡，因为Session问题会出现不停的提示要重新登录，这个问题其实是Session共享的问题，Nginx提供了集中Session共享的解决方案，这里我们使用的是nginx中的ip_hash技术，ip_hash能够将某个ip的请求定向到同一台后端，这样一来这个ip下的某个客户端和某个后端就能建立起稳固的session，ip_hash是在upstream配置中定义的：
+
+```
+upstream  doris.com {
+   server    172.22.197.238:8030 weight=3;
+   server    172.22.197.239:8030 weight=4;
+	 server    172.22.197.240:8030 weight=4;
```
Comment (morningman) [inline docs/zh-CN/faq/faq.md:329]:
```suggestion
    server    172.22.197.240:8030 weight=4;
```

File: docs/zh-CN/faq/faq.md [hunk 2]
```diff
@@ -318,6 +318,79 @@ failed to initialize storage reader. tablet=63416.1050661139.aa4d304e7a7aff9c-f0
...
+
+    # Load modular configuration files from the /etc/nginx/conf.d directory.
+    # See http://nginx.org/en/docs/ngx_core_module.html#include
+    # for more information.
+    include /etc/nginx/conf.d/*.conf;
+    #include /etc/nginx/custom/*.conf;
+    upstream  doris.com {
+        server    172.22.197.238:8030 weight=3;
+        server    172.22.197.239:8030 weight=4;
+	      server    172.22.197.240:8030 weight=4;
```
Comment (morningman) [inline docs/zh-CN/faq/faq.md:372]:
```suggestion
        server    172.22.197.240:8030 weight=4;
```

File: docs/en/faq/faq.md [hunk 3]
```diff
@@ -317,3 +317,77 @@ One situation is OVERCROWDED, which means that a large amount of unsent data at
...
+
+    # Load modular configuration files from the /etc/nginx/conf.d directory.
+    # See http://nginx.org/en/docs/ngx_core_module.html#include
+    # for more information.
+    include /etc/nginx/conf.d/*.conf;
+    #include /etc/nginx/custom/*.conf;
+    upstream  doris.com {
+        server    172.22.197.238:8030 weight=3;
+        server    172.22.197.239:8030 weight=4;
+	      server    172.22.197.240:8030 weight=4;
```
Comment (morningman) [inline docs/en/faq/faq.md:372]:
```suggestion
        server    172.22.197.240:8030 weight=4;
```

File: docs/en/faq/faq.md [hunk 4]
```diff
@@ -317,3 +317,77 @@ One situation is OVERCROWDED, which means that a large amount of unsent data at
...
+
+### Q22. Multiple FEs, when using Nginx to implement web UI load balancing, the login cannot be entered
+
+Doris can deploy multiple FEs. When accessing the Web UI, if you use Nginx for load balancing, you will be prompted to log in again because of Session problems. This problem is actually a session sharing problem. Nginx provides centralized session sharing. The solution, here we use the ip_hash technology in nginx, ip_hash can direct the request of a certain ip to the same backend, so that a certain client and a certain backend under this ip can establish a stable The session, ip_hash is defined in the upstream configuration:
+
+```
+upstream  doris.com {
+   server    172.22.197.238:8030 weight=3;
+   server    172.22.197.239:8030 weight=4;
+	 server    172.22.197.240:8030 weight=4;
```
Comment (morningman) [inline docs/en/faq/faq.md:329]:
```suggestion
    server    172.22.197.240:8030 weight=4;
```

**Comment by stalary** [comment]
I think this may lead to a problem that only one FE can be hit by all accesses in the company Intranet environment

**Pull request closed without merging**